src/gpuarray_buffer_opencl.c

```diff
@@ -113,6 +172,27 @@ cl_ctx *cl_make_ctx(cl_context ctx, int flags) {
     return NULL;
   }
   res->refcnt--; /* Prevent ref loop */
+
+  /* Create per-context OpenCL preamble */
+
+  // Create a dummy kernel and check GA_KERNEL_PROP_PREFLSIZE
+  rlk[0] = dummy_kern;
+  len = sizeof(dummy_kern);
+  // this dummy kernel does not require a CLUDA preamble
+  m = cl_newkernel((gpucontext *)res, 1, rlk, &len, "kdummy", 0, NULL, 0, &ret, NULL);
+  assert((m != NULL) && "Cannot create a dummy OpenCL kernel!");
```

Comment:
Don't use assert as error checking since the assert statements are removed in release mode.